src/wallet/rpcwallet.cpp

```diff
@@ -3496,7 +3500,7 @@ UniValue z_listunifiedreceivers(const UniValue& params, bool fHelp)
             "{\n"
             "  \"transparent\": \"address\",   (string) The legacy transparent address (P2PKH or P2SH)\n"
```

Comment:
```suggestion
            "  \"TRANSPARENT_TYPE\": \"address\",   (string) The legacy transparent address (either \"p2pkh\" or \"p2sh\", never both)\n"
```

Also alter the output accordingly.